setup.cfg

```diff
@@ -1,4 +1,5 @@
 [flake8]
+max-line-length: 120
```

Comment:
Makes sense. 80 seems absurdly small for current monitors. I'll submit a 120 PR for brainiak.